<?xml version="1.0"?>
<launch>

  <arg name="cf_num" default="1" />
  <arg name="z_coefficient" default="4"/>
  <arg name="continuous" default="True" />
  <arg name="air_vel" default=".5" />
  <arg name="land_vel" default=".25" />

  <node name="td_planner" pkg="planner" type="td_planner.py" output="screen">
  	<param name="cf_num" value="$(arg cf_num)"/>
  	<param name="z_coefficient" value="$(arg z_coefficient)" />
  </node>


</launch>